src/Signup.js

```diff
@@ -200,7 +207,7 @@ class Register extends Signup {
         return stage.complete().then(function(request) {
             if (request.auth) {
                 console.log("Stage %s is returning an auth dict", stageName);
-                return self._tryRegister(request.auth);
+                return self._tryRegister(request.auth, request.poll_for_success);
```

Comment:
I feel like there's an awful lot of stuff which isn't specced at all here. Do you have plans to spec this at some point?